app/services/conference_domains_service.rb

```diff
@@ -0,0 +1,27 @@
+class ConferenceDomainsService
```

Comment:
Tests should not touch the "real" internet.
Instead, you should mock the usage of `Resolv::DNS`.